README.md

```diff
@@ -40,3 +40,6 @@ You'll need a version of node >= 14 which can be downloaded from the [Node.js we
 
 ### Hosted Example
 See a [hosted version ](https://applepay-paypal-js-sdk.herokuapp.com)of the sample
+
+### PayPal Codespaces Links
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/paypal-examples/applepay)
```

Comment:
will need to add `Register your domain` step in readme or in codespaces configuration